### DISCUSSION TITLE
Customizing Card Fonts

### DISCUSSION THREAD

**Discussion by DigitalJens** [mscherotter/galeryst#27] [General]
Hi Micheal,

ist it possible to customize the card fonts like described here:
https://docstore.mik.ua/orelly/xml/xmlnut/ch12_07.htm

Or should the users better stay to the values like they are described in the cards tab (e.g. for compatibility reasons)?:
Fonts & weights
- museo-sans 300, 500, 700
- arboria 300, 400, 700

Best regards

Jens

**Comment by mscherotter** [comment]
You can also use web safe fonts listed here: https://www.w3schools.com/cssref/css_websafe_fonts.asp
